pyqir-generator/Cargo.toml

```diff
@@ -10,6 +10,7 @@ homepage = "https://github.com/qir-alliance/pyqir"
 repository = "https://github.com/qir-alliance/pyqir"
 
 [dependencies]
+inkwell = { git = "https://github.com/TheDan64/inkwell", branch = "master", default-features = false, features = ["target-x86"] }
 pyo3 = { version = "0.17.2", features = ["abi3-py37", "extension-module"] }
 qirlib = { path = "../qirlib" }
 
```

Comment:
I think there was a bad merge here. We see pyo3 dep from old branch instead of new base. Same with features block below.

Reply:
Sorry, I'm not sure what you mean. The only change to this file is the addition of the inkwell dependency, which is intentional.